thirdparty/build-thirdparty.sh

```diff
@@ -221,6 +221,9 @@ build_thrift() {
 
 # llvm
 build_llvm() {
+    export CFLAGS="-O3 -fno-omit-frame-pointer -std=c99 -D_POSIX_C_SOURCE=199309L"
```

Comment:
this is not the right way to change CFLAGS/CXXFLAGS, it will affect all other libraries build that follows this llvm build.

Reply:
Would it be correct if I add ﻿restore_compile_flags at the end of the function?

Reply:
yes.